robottelo/vm.py

```diff
@@ -249,6 +249,9 @@ def install_katello_agent(self):
             raise VirtualMachineError('Failed to install katello-agent')
         gofer_check = self.run('service goferd status')
         if gofer_check.return_code != 0:
+            self.run('service goferd start')
```

Comment:
Frankly i'm hoping while this PR gets 2 acks and all the issues resolved, katello-agent BZ will become fixed (as it's in progress right now) and i'll just remove these lines 😄 
Otherwise i'd like to insist on adding a workaround here in any acceptable way, as it blocks too much tests for us, even those which aren't directly related (like capsule or content management tests).